package.json

```diff
@@ -85,18 +85,21 @@
     "lint:production:lit-analyzer": "lit-analyzer **/*.ts",
     "lint:production:stylelint": "tsc --noCheck --outDir ./dist && stylelint '**/*.styles.ts' --custom-syntax postcss-lit --no-color",
     "test": "per-env",
-    "test:development": "npm-run-all --parallel test:development:components test:development:serve-component-coverage",
-    "test:development:components": "web-test-runner --watch",
-    "test:development:lint-rules": "vitest --config ./vitest.config.js & chokidar './src/eslint/**' './src/stylelint/**' --ignore './src/eslint/rules/*.test.ts' './src/stylelint/*.test.ts' --initial --silent --command 'tsc --noCheck --outDir ./dist'",
-    "test:development:lint-rules:comment": "Excluded from `test:development` because Vitest muddies the console when running alongside Web Test Runner.",
-    "test:development:serve-component-coverage": "npx http-server dist/coverage/lcov-report --silent",
-    "test:production": "pnpm start:production:stylesheets && npm-run-all --aggregate-output --print-label --parallel test:production:*",
-    "test:production:lint-rules": "tsc --noCheck --outDir ./dist && vitest run --config ./vitest.config.js",
-    "test:production:components": "web-test-runner",
+    "test:development": "npm-run-all --parallel test:development:web-test-runner test:development:web-test-runner:serve-coverage",
+    "test:development:playwright": "playwright test --quiet --only-changed",
```

Comment:
Adding this script threw everything into disarray because both it and `test:development:components` test components. So what to name this command?

So back to test scripts named after the test runner. I the non-test scripts to match:  `start:production:components` → `start:production:typescript`. Now every script fits in nicely and is named after the technology it uses as opposed to the thing it does.

Reply:
The reason for the script is that, even though snapshots aren't generated locally, it turned out to be useful to still run Playwright when writing tests.

For example, [this](https://github.com/CrowdStrike/glide-core/pull/740#discussion_r1978038233) is something that would have been caught in CI (or beforehand on push) if this script existed from the start.